package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "peerDependencies": {
+    "lodash.throttle": "^4.1.1",
```

Comment:
Is there a reason you didn't commit changes to package.lock.json?

Reply:
This is a peer dependency